packages/angular/src/generators/ng-add/files/decorate-angular-cli/decorate-angular-cli.js__tmpl__

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
There's a copy of this in `packages/workspace/src/generators/utils/decorate-angular-cli.js__tmpl__`. Do we need this?

I think we should deep import this from `@nrwl/angular` for the preset.

Reply:
Yeah, I agree. I'll update the preset to deep import this template here.

Reply:
Actually, we can't because we'd be introducing a dependency to `@nrwl/angular`.

Reply:
The options I see:

- In `@nrwl/workspace:workspace` deep import from `@nrwl/angular`. It will introduce a dependency to `@nrwl/angular` which I don't think is something we want.
- In `@nrwl/angular:ng-add` deep import from `@nrwl/workspace`. It will require adding `@nrwl/workspace` as a dependency or having the generator install it before running the logic of the generator.

I think we should be better with the second option. Even though it feels like decorating the angular cli should be a concern for the Angular plugin, we actually support choosing that cli with any type of workspace, so that makes it a concern of any workspace. It wouldn't be wrong to have `@nrwl/workspace` as a dependency for `@nrwl/angular`.